test/functional/services/browser.ts

```diff
@@ -199,6 +199,14 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
       return await driver.get(url);
     }
 
+    /**
+     * Pauses the execution in the browser, similar to setting a breakpoint for debugging.
+     * @return {Promise<void>}
+     */
+    public async pause() {
+      await driver.executeAsyncScript(`(async () => { debugger; return Promise.resolve(); })()`);
```

Comment:
When/If this turns out to be useful, can you add a note in some developer docs about how to use this?

Reply:
Ah! Great reminder, thanks!